docs/nitclk/class.rst

```diff
@@ -460,17 +460,17 @@ exported_sync_pulse_output_terminal
 
         The following table lists the characteristics of this property.
 
-            +----------------+------------+
-            | Characteristic | Value      |
-            +================+============+
-            | Datatype       | str        |
-            +----------------+------------+
-            | Permissions    | read-write |
-            +----------------+------------+
-            | Channel Based  | No         |
-            +----------------+------------+
-            | Resettable     | No         |
-            +----------------+------------+
+            +----------------+-------------+
+            | Characteristic | Value       |
+            +================+=============+
+            | Datatype       | default_str |
```

Comment:
this is wrong

Reply:
Lost fix when moving to new PR. Fixed. Again.